packages/open-payments/src/client/index.ts

```diff
@@ -125,7 +126,8 @@ export const createAuthenticatedClient = async (
     grant: createGrantRoutes({
       axiosInstance,
       openApi: authServerOpenApi,
-      logger
+      logger,
+      client: args.client
```

Comment:
I was going to have `createAuthenticatedClient` query `${client}/jwks.json` to validate that the key with the specified `keyid` is served there, but `backend` calls `createAuthenticatedClient` long before its routes are set up.

Reply:
Yeah I was thinking you'd possibly get some kind of race condition type behaviour - could add some kind of retry mechanism but gets too complicated for what it's worth. 

 Is there a more straightforward name than `client`? Even just `paymentPointerUrl`? or `clientPaymentPointer`?